### ISSUE TITLE
FetchError: request to https://api.expo.dev/v2/manifest/sign failed, reason: read ECONNRESET

### ISSUE THREAD

**Issue by pullmann4rent** [expo/expo#22377]
### Summary

Hello,

I want to start my server but I got this error:

`FetchError: request to https://api.expo.dev/v2/manifest/sign failed, reason: read ECONNRESET`

### What platform(s) does this occur on?

iOS

### SDK Version

48

### Environment

  expo-env-info 1.0.5 environment info:
    System:
      OS: Windows 10 10.0.19044
    Binaries:
      Node: 16.13.2 - C:\Program Files\nodejs\node.EXE
      Yarn: 1.22.11 - ~\AppData\Roaming\npm\yarn.CMD  
      npm: 8.1.2 - C:\Program Files\nodejs\npm.CMD    
    IDEs:
      Android Studio: Version     2020.3.0.0 AI-203.7717.56.2031.7583922
    npmPackages:
      expo: ^48.0.0 => 48.0.15
      react: 18.2.0 => 18.2.0
      react-dom: 18.2.0 => 18.2.0
      react-native: 0.71.7 => 0.71.7
      react-native-web: ~0.18.9 => 0.18.10
    Expo Workflow: managed

### Minimal reproducible example

npx expo start

**Comment by alickcai** [comment]
I have same issue

FetchError: request to https://api.expo.dev/v2/manifest/sign failed, reason: connect ETIMEDOUT 104.18.4.104:443

**Comment by alickcai** [comment]
More info attached:

FetchError: request to https://api.expo.dev/v2/versions/latest failed, reason: connect ETIMEDOUT 104.18.5.104:443
    at ClientRequest.<anonymous> (D:\szrncode\sfRNcodes\node_modules\node-fetch\lib\index.js:1491:11)
    at ClientRequest.emit (node:events:527:28)
    at TLSSocket.socketErrorListener (node:_http_client:454:9)
    at TLSSocket.emit (node:events:527:28)
    at emitErrorNT (node:internal/streams/destroy:157:8)

**Comment by alickcai** [comment]
Above issue happened for both Android and iOS devices

**Comment by Kudo** [comment]
@pullmann4rent wondering whether this case still happening to you?

**Comment by Kudo** [comment]
how about this command? `curl.exe --verbose -I https://api.expo.dev/v2/versions/latest`

**Comment by Q8root** [comment]
> how about this command? `curl.exe --verbose -I https://api.expo.dev/v2/versions/latest`

q8root@Ahmads-Air ~ % curl --verbose -I https://api.expo.dev/v2/versions/latest
*   Trying 104.18.4.104:443...
* Connected to api.expo.dev (104.18.4.104) port 443 (#0)
* ALPN: offers h2
* ALPN: offers http/1.1
*  CAfile: /etc/ssl/cert.pem
*  CApath: none
* (304) (OUT), TLS handshake, Client hello (1):
* (304) (IN), TLS handshake, Server hello (2):
* (304) (IN), TLS handshake, Unknown (8):
* (304) (IN), TLS handshake, Certificate (11):
* (304) (IN), TLS handshake, CERT verify (15):
* (304) (IN), TLS handshake, Finished (20):
* (304) (OUT), TLS handshake, Finished (20):
* SSL connection using TLSv1.3 / AEAD-CHACHA20-POLY1305-SHA256
* ALPN: server accepted h2
* Server certificate:
*  subject: C=US; ST=California; L=San Francisco; O=Cloudflare, Inc.; CN=sni.cloudflaressl.com
*  start date: Oct 16 00:00:00 2022 GMT
*  expire date: Oct 16 23:59:59 2023 GMT
*  subjectAltName: host "api.expo.dev" matched cert's "*.expo.dev"
*  issuer: C=US; O=Cloudflare, Inc.; CN=Cloudflare Inc ECC CA-3
*  SSL certificate verify ok.
* Using HTTP2, server supports multiplexing
* Copying HTTP/2 data in stream buffer to connection buffer after upgrade: len=0
* h2h3 [:method: HEAD]
* h2h3 [:path: /v2/versions/latest]
* h2h3 [:scheme: https]
* h2h3 [:authority: api.expo.dev]
* h2h3 [user-agent: curl/7.86.0]
* h2h3 [accept: */*]
* Using Stream ID: 1 (easy handle 0x153810a00)
> HEAD /v2/versions/latest HTTP/2
> Host: api.expo.dev
> user-agent: curl/7.86.0
> accept: */*
> 
* Connection state changed (MAX_CONCURRENT_STREAMS == 256)!
< HTTP/2 200 
HTTP/2 200 
< date: Tue, 16 May 2023 18:46:07 GMT
date: Tue, 16 May 2023 18:46:07 GMT
< content-type: application/json; charset=utf-8
content-type: application/json; charset=utf-8
< content-length: 27419
content-length: 27419
< vary: Accept-Encoding, Origin
vary: Accept-Encoding, Origin
< x-frame-options: SAMEORIGIN
x-frame-options: SAMEORIGIN
< x-content-type-options: nosniff
x-content-type-options: nosniff
< etag: "6b1b-Y3src2lSujzqPAtYDQ5ubLW8WZo"
etag: "6b1b-Y3src2lSujzqPAtYDQ5ubLW8WZo"
< via: 1.1 google
via: 1.1 google
< strict-transport-security: max-age=31536000; includeSubDomains
strict-transport-security: max-age=31536000; includeSubDomains
< alt-svc: h3=":443"; ma=86400, h3-29=":443"; ma=86400
alt-svc: h3=":443"; ma=86400, h3-29=":443"; ma=86400
< cf-cache-status: DYNAMIC
cf-cache-status: DYNAMIC
< server: cloudflare
server: cloudflare
< cf-ray: 7c85c818abed746c-LHR
cf-ray: 7c85c818abed746c-LHR

< 
* Connection #0 to host api.expo.dev left intact

I got the error when open a project in android studio

**Comment by yiday** [comment]
Having same issue
FetchError: request to https://api.expo.dev/v2/manifest/sign failed, reason: Socket connection timeout

**Comment by Kudo** [comment]
hi there! how about testing another endpoint?
`curl --verbose -I https://api.expo.dev/v2/manifest/sign` and also check testing the url `https://api.expo.dev/v2/manifest/sign` in browsers.

**Comment by Davidon4** [comment]
Ok, How I solve this issue was, I figured the latest nodejs v18 was not working with latest expo sdk 48. I downgraded it to v16.4.0 and then i did "npm i -g expo-cli" inside my project after the installation. "npx expo start" voila it worked.

**Comment by skjortan23** [comment]
Hm getting the same error and upgrading od downgrading node does not seem to help.
starting via `npx expo start`

i am on latest version and node 16

**Comment by skjortan23** [comment]
For anyone having this issue. I cant connect to api.expo.dev from any ipv6 address jumping on vpn that exits via an ipv4 it seems to work. cc @Kudo or anyone from the expo team.

So this seems to be ipv6 related

**Comment by splenwilz** [comment]
This issue may arise when you uninstall a library that is somehow related to Expo. At least for my specific use case, that is what happened. I had initially installed ```react-native-dropdown-picker``` but later discovered that it wasn't necessary for my project. So, I uninstalled it, and when I tried to run my app with ```npm start,``` I encountered this error. To resolve this, I had to globally install ```Expo CLI``` again using the following command:

```npm i -g expo-cli```

Once I did that, everything worked smoothly.
